1-JavaScript/Week2/homework/ex7-mindPrivacy.js

```diff
@@ -30,8 +30,13 @@ const employeeRecords = [
 ];
 
 // ! Function under test
-function filterPrivateData(/* TODO parameter(s) go here */) {
-  // TODO complete this function
+function filterPrivateData(employees) {
+  let publicInfo=[];
+for(let employee of employees){
+  const {name, occupation ,gender , email, salary}=employee;
```

Comment:
Nice one, destructuring 💪 However, you don't need to grab the properties that you are not going to use: gender and salary.

```suggestion
  const {name, occupation, email} = employee;
```

Try and use the `.map()` method for this assignment, then you also don't need to make a new variable `publicInfo`. This array method is especially made this scenarios like this where you want to return a new array and make some modifications. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map